src/js/disability-benefits/actions/index.js

```diff
@@ -46,6 +46,23 @@ export function getClaims() {
   };
 }
 
+export function getFilteredClaims(open = true) {
+  return (dispatch) => {
+    makeAuthRequest('/v0/disability_claims',
+      null,
+      dispatch,
+      claims => {
+        const filteredClaims = claims.data.filter(function(claim) {
+          return claim.attributes.open === open;
+        });
+        dispatch({ type: SET_CLAIMS, claims: filteredClaims, meta: claims.meta });
+        dispatch({ type: CHANGE_CLAIMS_PAGE, page: 1 });
```

Comment:
This resets the page when switching tabs, which is intended behavior. However, it also resets the page when going back to the list from an individual claim. Is there a better way to achieve the reset on tab switch but keep current page accurate when going back?